projects/common/src/utilities/formatters/enum/display-string-enum.ts

```diff
@@ -5,5 +5,9 @@ export const displayStringEnum = (provided?: string): string => {
     return '-';
   }
 
-  return startCase(provided.toLowerCase());
+  // This removes dashes and underscores and gives all words initial caps
+  const startCased = startCase(provided);
+
+  // We only want the first word capitalized.
```

Comment:
If that's what we want, what about `capitalize(lowerCase(provided))` - believe that works the same way as startCase:
```
_.lowerCase('--Foo-Bar--');
// => 'foo bar'
```

Reply:
That does look cleaner, but I think this one fails for camelCase and PascalCase.

Reply:
![image](https://user-images.githubusercontent.com/45047841/113938345-566d3280-97af-11eb-8dcc-15ccd451c416.png)

Reply:
lodash should have ng pipes for each method. `'text' | lowercase | capitalize`